components/Grid/Index.vue

```diff
@@ -0,0 +1,364 @@
+<template>
+  <div class="w-[1020px] h-[115px] rounded-[3px] text-[10px] border flex">
```

Comment:
1. `w-[1020px]` and `h-[115px]` are static styles. change it to `w-full` and `h-fit`
2. change `text-[10px]` to `text-xs`
3. change `rounded-[3px]` to `rounded-xs`